components/link/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
```

Comment:
Does this need to be in here? why can't we use the core version of `<Link />` to start?